pyccel/parser/grammar/openmp.tx

```diff
@@ -42,11 +42,11 @@ OmpConstructOrDirective:
 ////////////////////////////////////////////////////
 OmpParallelConstruct:   'parallel'        combined=ParallelCombined? clauses*=AllOmpClauses;
 OmpLoopConstruct:       'for'             combinedsimd='simd'? clauses*=AllOmpClauses;
-OmpTaskLoopConstruct:   name='taskloop'   clauses*=AllOmpClauses;
+OmpTaskLoopConstruct:   'taskloop'   clauses*=AllOmpClauses;
 OmpSingleConstruct:     'single'          clauses*=AllOmpClauses;
 OmpCriticalConstruct:   name='critical'   clauses*=AllOmpClauses;
 OmpMaskedConstruct:     'masked'          combined=MaskedCombined? clauses*=AllOmpClauses;
-OmpSimdConstruct:       name='simd'       clauses*=AllOmpClauses;
+OmpSimdConstruct:       'simd'       clauses*=AllOmpClauses;
 OmpAtomicConstruct:     name='atomic'     clauses*=AllOmpClauses; // Not all clauses were added (can't test them).
 OmpTaskConstruct:       name='task'       clauses*=AllOmpClauses;
 OmpCancelConstruct:     name='cancel'     clauses*=OmpCancelType;
```

Comment:
Could somebody explain why some constructs use `name='STRING'` and others just `'STRING'`?

Reply:
@nhamidn

Reply:
Constructs that don't accept any clauses like `#$omp master`, if it's grammar is : `OmpMasterConstruct:   'master'; ` will generate an error at this line https://github.com/pyccel/pyccel/blob/1621df5c8786fdaba5ec49b956a0ec980bb9c777/pyccel/parser/syntax/openmp.py#L863  saying `AttributeError: 'str' object has no attribute 'expr'`, and this is because textx will not create any object for this construct since we don't save any value in the grammar and will just return a string.

the `name= ` can be removed from the construct that save at least one value different than `name`.